.github/workflows/ci.yml

```
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+    - name: Install pre-commit
+      run: |
+        pip install pre-commit
+    - name: Run pre-commit
+      run: |
+        pre-commit run --all-files
```

```suggestion
        pre-commit run --all-files --color always --verbose
```

Suggesting this so that debugging is a bit easier. You might also want to consider caching pre-commit using https://github.com/marketplace/actions/cache to speed things up.